tests/e2e/specs/productDetailPage_spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Product detail page', () => {
+  before(() => {
+    cy.visit('/product/lemare-booties-0778-grey/M0E20000000E0WX');
+  });
+
+  it('Displays a single product details', () => {
+    cy.get('[data-test=product-description]', { timeout: 20000 })
+      .then(($product) => {
+        cy.wrap($product)
+          .find('[data-test=pdp-product-title]')
+          .should('contain', 'Booties Lemare grey');
+
+        cy.wrap($product)
+          .find('[data-test=product-sku]')
+          .should('contain', 'M0E20000000E0WX');
+
+        cy.wrap($product)
+          .find('[data-test=product-discounted-price]')
+          .should('contain', '248,75 €');
+        cy.wrap($product)
+          .find('[data-test=product-discount-price]')
+          .should('contain', '174,12 €');
+
+        cy.wrap($product)
+          .find('[data-test=panel-default]')
+          .should('exist')
+          .find('[data-test=accordion-toggle]')
+          .click();
+
+        cy.wrap($product)
+          .find('[data-test=list-attributes]')
+          .should('have.length', 7)
+          .eq(2)
+          .should('contain', 'size:')
```

Comment:
I guess you can replace `contain` by `eq`, since it is exactly what you wrote there and not a substring.